svc.html

```diff
@@ -0,0 +1,347 @@
+// High resolution base layer has half the input framerate, full resolution
+// Temporal enhancement layers have full input framerate
+var encodings = [
+  {rid: 'H', encodingId: 'H0', scaleResolutionDownBy: 2.0, scaleFramerateDownBy: 2.0},
+  {rid: 'F', encodingId: 'F0', scaleFramerateDownBy: 2.0}, 
+  {rid: 'H', encodingId: 'H1', scaleResolutionDownBy: 2.0, dependencyEncodingIds: ['H0']},
+  {rid: 'F', encodingId: 'F1', dependencyEncodingIds: ['F0']}
+];
+                    </pre>
+          <p>Below is a representation of 2-layer temporal scalability combined with
+          2-layer spatial simulcast. Solid arrows represent temporal prediction. In the
+          diagram, I0 is the base-layer I-frame, and P0 represents base-layer P-frames.
+          EI0 is an enhanced resolution base-layer I-frame, and EP0 represents P-frames
+          within the enhanced resolution base layer. P1 represents the first temporal
+          enhancement layer, and EP1 represents a temporal enhancement to the enhanced
+          resolution simulcast base-layer.</p>
+          <figure>
+            <img alt="2-layer spatial simulcast and temporal scalability encoding" src=
+            "images/2-layer-spatialsim-temporal.svg" style="width:75%">
+            <figcaption>
+              2-layer spatial simulcast and temporal scalability encoding
+            </figcaption>
+          </figure>
+        </section>
+        <section class="informative" id="rtcrtpencodingspatialscal-example*">
+          <h4>Spatial Scalability</h4>
+          <pre class="example highlight">
+// Example of 3-layer spatial scalability encoding with the base layer having
+// one quarter input resolution and enhancement layers yielding one half and
+// full resolution
+var encodings = [
+  {encodingId: 'q', scaleResolutionDownBy: 4.0},
+  {encodingId: 'h', scaleResolutionDownBy: 2.0, dependencyEncodingIds: ['q']},
+  {encodingId: 'f', dependencyEncodingIds: [['q', 'h']}
+]
+
+// Example of 3-layer spatial scalability with all but the base layer disabled
+var encodings = [
+  {encodingId: 'q', scaleResolutionDownBy: 4.0, active: true},
+  {encodingId: 'h', scaleResolutionDownBy: 2.0, active: false},
+  {encodingId: 'f', active: false}
+]
+
+// Example of 2-layer spatial scalability combined with 2-layer temporal scalability
+// Base layer has one half input framerate and half resolution
+// Temporal enhancement layer has full input framerate, half resolution
+// Spatial enhancement to the base layer has half input framerate, full resolution
+// Temporal enhancement to the spatial enhancement layer has full framerate and resolution
+var encodings = [
+  {encodingId: 'H0', scaleResolutionDownBy: 2.0, scaleFramerateDownBy: 2.0}, 
+  {encodingId: 'H1', scaleResolutionDownBy: 2.0, dependencyEncodingIds: ['H0']},
+  {encodingId: 'F0', scaleFramerateDownBy: 2.0, dependencyEncodingIds: ['H0']},
+  {encodingId: 'F1', dependencyEnodingIds: ['F0', 'H1']}
```

Comment:
Even more combinations of dependency and weird values, what are valid and what are invalid setups? What happens if I do setParameters() with weird values?